codex-rs/app-server/tests/suite/v2/account.rs

```diff
@@ -304,6 +307,29 @@ async fn login_account_chatgpt_start() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn cancel_login_account_not_found_returns_status() -> Result<()> {
+    let codex_home = TempDir::new()?;
+    create_config_toml(codex_home.path(), CreateConfigTomlParams::default())?;
+
+    let mut mcp = McpProcess::new(codex_home.path()).await?;
+    timeout(DEFAULT_READ_TIMEOUT, mcp.initialize()).await??;
+
+    let request_id = mcp
+        .send_cancel_login_account_request(CancelLoginAccountParams {
+            login_id: Uuid::new_v4().to_string(),
+        })
+        .await?;
+    let resp: JSONRPCResponse = timeout(
+        DEFAULT_READ_TIMEOUT,
+        mcp.read_stream_until_response_message(RequestId::Integer(request_id)),
+    )
+    .await??;
+    let response: CancelLoginAccountResponse = to_response(resp)?;
+    assert_eq!(response.status, CancelLoginAccountStatus::NotFound);
+    Ok(())
+}
+
 #[tokio::test]
 // Serialize tests that launch the login server since it binds to a fixed port.
 #[serial(login_port)]
```

Comment:
If the above test also requires the login port, it should also have this `#[serial(login_port)]` annotation?